app/views/community/patients/_form.html.erb

```diff
@@ -153,30 +161,23 @@
                        autocomplete: "sus", required: false,
                        class: "form-control" %>
     <small id="susHelp" class="form-text text-muted">
-      Número do cartão SUS. Descubra o número acessando
-      <a href="https://cartaosusdigital.saude.gov.br" target="cartaosus">Cartão SUS Digital</a>.
+      Número do cartão SUS. Descubra o número acessando o
+      <a href="https://conectesus-paciente.saude.gov.br" target="cartaosus">ConecteSUS</a>.
     </small>
   </div>
 </div>
 
-<script type="text/javascript">
-  $(function () {
-    $('[data-group-parent]').on('change', function () {
-      groupToggle($(this))
-    }).each(function () {
-      groupToggle($(this))
-    })
-
-    let selectors = [
-      '[data-group-context-priority] :checkbox',
-      '[data-group-context-comorbidity] :checkbox'
-    ];
-    selectors.forEach(function (selector) {
-      $(selector).on('change', function () {
-        requireToggle($(selector))
-      }).each(function () {
-        requireToggle($(selector))
-      })
-    })
-  })
-</script>
+<% if InquiryQuestion.active.any? -%>
+  <h3>
+    Inquérito
+    <%= optional_field_tag %>
+  </h3>
+
+  <%= embedded_page :patient_inquiry_intro -%>
```

Comment:
Não entendi o porquê, mas tive que mudar o `def embedded_page` pra forçando desse jeito:

```
def embedded_page(path)
  Page.find_by!(path: 'patient_inquiry_intro').html
end
```

quando subi o app e fazia get na `/` o path que chegava nessa função era o `:home` aí não achou na relação `Page` talvez eu não tenha rodado alguma coisa

Reply:
@pedroCervi acredito que que faltou rodar `rake db:seed:development` para popular as paginas.